mock-web-services/app.py

```diff
@@ -0,0 +1,31 @@
+from flask import Flask, Response, request, json
+from esri import EsriMockWebService
+
+app = Flask(__name__)
+esriWS = EsriMockWebService(app)
+
+
+@app.route("/esri/findAddressCandidates")
+def findAddressCandidates():
+    try:
+        address = request.args.get("address")
+        return esriWS.findAddressCandidates(address)
+    except Exception as e:
+        print(e)
+        return Response("Bad request", status=400)
+
+
+@app.route("/esri/serviceAreaFeatures")
+def serviceAreaFeatures():
```

Comment:
**nitpick:** snake_case for method names in this file

Reply:
Thanks. It gets confusing going back and forth sometimes between java and python.